src/components/tables/Accounts.tsx

```diff
@@ -72,24 +68,20 @@ function AccountsTable() {
                         </TableRow>
                     </TableHead>
                     <TableBody>
-                        {loading ? (
+                        {isLoading ? (
```

Comment:
We should probably touch base on what variable prefix and postfix patterns would like to adopt soon. Using the highlighted code as an example, I prefer for variables to be written in an active voice (e.g. _loading_ as opposed to _isLoading_). When a variable is involved in a conditional expression, the expression is naturally interpreted as a question, therefore the prefix results in a wordy translation (e.g., _if loading, do x_ as opposed to _if is loading, do x_). Just my two cents.